docs/sb_newBeamtime.rst

```diff
@@ -10,11 +10,7 @@ Goals of the Setup
  2. Create empty directories with the correct directory structure
  3. Prepopulate the directories with configuration files as needed
  4. Initialize the metadata stack (the list of metadata) with beamtime level information
- 5. Make certain that the proxy is running see the
-
-`proxy documentation in bluesky <http://nsls-ii.github.io/bluesky/callbacks.html#minimal-example>`_ (make certain ``bluesky-0MQ-proxy 5577 5578`` is running)
-
-If the proxy is not running run ``source activate 17Q3-dev`` then ``bluesky-0MQ-proxy 5577 5578``
+ 5. Make certain that the proxy is running. `proxy documentation in bluesky <http://nsls-ii.github.io/bluesky/callbacks.html#minimal-example>`_ (make certain ``bluesky-0MQ-proxy 5577 5578`` is running)
 
```

Comment:
would it be good to turn inline code highlighting into sphinx code block so that it's easier to follow?
(as bluesky  doc does)